backends/capabilities_android/java/org/haxe/duell/capabilities/Capabilities.java

```diff
@@ -72,6 +75,22 @@ public static double getDensity()
         return metrics.density;
     }
 
+    public static double getTotalMemory()
+    {
+        MemoryInfo mi = new MemoryInfo();
+        ActivityManager activityManager = (ActivityManager) DuellActivity.getInstance().getSystemService(Context.ACTIVITY_SERVICE);
+        activityManager.getMemoryInfo(mi);
+
+        if (android.os.Build.VERSION.SDK_INT > android.os.Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
my mistake I made a commit that fixes the issue, the totalMem is only available for api level 16 or greater it is now working properly.